src/ui/server/helpers/task-list/generate-groups-and-tasks/prepare-application.ts

```diff
@@ -53,7 +53,7 @@ const createPrepareApplicationTasks = (referenceNumber: number, otherGroups: Tas
   };
 
   const EXPORTER_BUSINESS = {
-    href: '#',
+    href: `${INSURANCE_ROOT}/${referenceNumber}${EXPORTER_BUSINESS_ROUTES.COMPANY_DETAILS}`,
```

Comment:
Nice one! Not for this PR but now that we have this, we can update E2E tests to reflect the full journey e.g

- click on the link in task list
- on X page

Also saves us doing "manually navigate to the page" kind of things in cypress